frontend/src/views/Results.tsx

```diff
@@ -1,25 +1,62 @@
-import React, { useEffect, useState } from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import styled from 'styled-components';
+import copy from 'copy-to-clipboard';
 import { useLocation, useHistory } from 'react-router-dom';
 import { Message } from 'stompjs';
-import { LargeText, Text } from '../components/core/Text';
+import { LargeText } from '../components/core/Text';
 import {
   getGame, Game, Player, playAgain,
 } from '../api/Game';
-import { checkLocationState } from '../util/Utility';
+import { checkLocationState, leaveRoom } from '../util/Utility';
 import { errorHandler } from '../api/Error';
 import PlayerResultsCard from '../components/card/PlayerResultsCard';
-import { PrimaryButton } from '../components/core/Button';
+import { PrimaryButton, SecondaryRedButton } from '../components/core/Button';
 import ErrorMessage from '../components/core/Error';
 import Loading from '../components/core/Loading';
 import {
   connect, disconnect, routes, subscribe,
 } from '../api/Socket';
 import { User } from '../api/User';
-import { ThemeConfig } from '../components/config/Theme';
+import Podium from '../components/special/Podium';
+import { HoverContainer, HoverElement, HoverTooltip } from '../components/core/HoverTooltip';
+import { Coordinate } from '../components/special/FloatingCircle';
+import {
+  CopyIndicator,
+  CopyIndicatorContainer,
+  InlineCopyIcon,
+} from '../components/special/CopyIndicator';
 
 const Content = styled.div`
-  padding: 0 20%;
+  padding: 0;
+`;
+
+const PrimaryButtonHoverElement = styled(HoverElement)`
+  width: 10rem;
+  height: 2.75rem;
+  top: 1.2rem;
+  left: 1.2rem;
+`;
+
+const PodiumContainer = styled.div`
+  display: flex;
+  justify-content: center;
+`;
+
+const InviteContainer = styled.div`
+  width: 70%;
+  margin: 20px auto 0 auto;
+  border-radius: 5px;
+  box-shadow: 0 1px 4px rgba(0, 0, 0, 0.24);
+  padding: 5px;
```

Comment:
nit: Move this padding to the styling text instead, or move the "onclick" function to the styling container. The main reason is that this creates a padding of 5px where clicking does nothing.